.github/vale/dicts/klaw.dic

```diff
@@ -0,0 +1,61 @@
+60
```

Comment:
60? :o

Also, did you do this manually? Amazing haha!

Reply:
Yes :D But it was not too bad, I ran the check, copied the console output, took all words that where marked as error and added the ones where it's a correct word.